packages/ove-app-audio/README.md

```diff
@@ -1,20 +1,45 @@
-# Audio App 
+# Audio App
 
-This app supports the playing of audio files within the OVE Framework.
+This app supports the playing of audio files within the OVE Framework. The audio app is powered by the [Howler.js](https://howlerjs.com/) audio library, thus it supports Web Audio API by default and falls back to HTML5 Audio if required. This app is intended to provide a distributed layer on top of the Howler Library.  

```

Comment:
~~The audio app~~ **It** is powered by the [Howler.js](https://howlerjs.com/) audio library, ~~thus it~~ **so** supports **the** Web Audio API by default and falls back to HTML5 Audio if required.